scripts/start.ts

```diff
@@ -42,10 +42,10 @@ function processTypeInNestedObject(obj: any) {
 
 for await (const entry of walk(".")) {
   if (entry.isFile) {
-    if (entry.path.endsWith(OPENAPI_EXTENSION)) {
+    if (entry.path.endsWith(OPENAPI_EXTENSION) && (entry.path === "vtex\\utils\\openapi\\my.openapi.json")) {
```

Comment:
Should be here?

Reply:
I don't know why but path doesn't recognize a new entry and doesn't generate a gen.api